src/Exception/BadConfigValueAlongsideProvider.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\AdminBundle\Exception;
+
+use InvalidArgumentException;
+
+/**
+ * @author Marko Kunic <kunicmarko20@gmail.com>
+ */
+final class BadConfigValueAlongsideProvider extends InvalidArgumentException implements SonataException
```

Comment:
It is more optimal to extend with a `\` like this:
`final class BadConfigValueAlongsideProvider extends \InvalidArgumentException implements SonataException`
To spare php engine from not looking up the class in the current namespace first.

Reply:
`\` or `use` statement are the same thing, personally, I do not like seeing `\` in my code.